fuzz-testing/src/main/scala/org/apache/comet/fuzz/Meta.scala

```diff
@@ -108,6 +124,8 @@ object Meta {
 
   val unaryArithmeticOps: Seq[String] = Seq("+", "-")
 
-  val binaryArithmeticOps: Seq[String] = Seq("+", "-", "*", "/", "%", "&", "|", "^")
+  val binaryArithmeticOps: Seq[String] = Seq("+", "-", "*", "/", "%", "&", "|", "^", "<<", ">>")
+
+  val comparisonOps: Seq[String] = Seq("=", "<=>", ">", ">=", "<", "<=")
```

Comment:
FYI there is a known `0.0 != -0.0` issue https://github.com/apache/datafusion/issues/11108